internal/cmdget/cmdget.go

```diff
@@ -87,8 +88,9 @@ func (r *Runner) runE(c *cobra.Command, args []string) error {
 	}
 
 	if r.AutoSet {
-		if err := setters.PerformAutoSetters(r.Get.Destination); err != nil {
-			return err
+		if err := setters.PerformAutoSetters(r.Get.Destination, filepath.Dir(r.Get.Destination)); err != nil {
+			// do nothing as auto-setters are best effort and should not throw an error if failed
```

Comment:
We shouldn't swallow any errors here.

Reply:
Done. Printed error messages to stdout for each failure and continued. Fatal errors are bubbled up.